src/Microsoft.Azure.ServiceBus/Core/IReceiverClient.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Core
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface IReceiverClient : IClientEntity
```

Comment:
Sometimes writing the doco along with the code raises good questions. Not going to insist on this, but will remind from time to time 😉